test/functional/services/remote/remote.js

```diff
@@ -1,18 +1,24 @@
 import { initLeadfootCommand } from './leadfoot_command';
 import { createRemoteInterceptors } from './interceptors';
-import { ChromedriverApi } from './chromedriver_api';
+import { BrowserdriverApi } from './browserdriver_api';
 
 export async function RemoteProvider({ getService }) {
   const lifecycle = getService('lifecycle');
   const config = getService('config');
   const log = getService('log');
+  const possibleBrowsers = ['chrome', 'firefox'];
+  let browserType = process.env.TEST_BROWSER_TYPE || 'chrome';
 
-  const chromedriverApi = await ChromedriverApi.factory(log, config.get('chromedriver.url'));
-  lifecycle.on('cleanup', async () => await chromedriverApi.stop());
+  if (!possibleBrowsers.includes(browserType)) {
+    browserType = 'chrome';
+  }
 
-  await chromedriverApi.start();
+  const browserdriverApi = await BrowserdriverApi.factory(log, config.get(browserType + 'driver.url'), browserType);
```

Comment:
I think `browserDriverApi` would be more correct casing. `chromedriver` was all lowercase because that's how chrome does things, but we're not bound to that anymore.

Reply:
Likewise with `BrowserDriverApi`.

Reply:
Fixed.